docs/user-guide/deploying-your-docs.md

```diff
@@ -43,34 +43,24 @@ files locally.
 
 ### Organization and User Pages
 
-User and Organization Pages sites are not tied to a specific project, and the
-site files are deployed to the `master` branch in a dedicated repository named
-with the GitHub account name. Therefore, you need working copies of two
-repositories on our local system. For example, consider the following file
-structure:
-
-```no-highlight
-my-project/
-    mkdocs.yml
-    docs/
-orgname.github.io/
-```
+GitHub Pages also allows users to create User and Organization Pages sites
+that are not tied to a specific project. Rather, the site files are deployed to
+the `master` branch in a dedicated repository named with the GitHub account
+name, i.e. `<user>.github.io` or `<org>.github.io`.
 
-After making and verifying updates to your project you need to change
-directories to the `orgname.github.io` repository and call the
-`mkdocs gh-deploy` command from there:
+After making and verifying updates to a dedicated user or organization
+repository on its `master` branch, call `mkdocs gh-deploy` in the
+command line and specify the site type with the `--page-type` option.
+Inputs for this option include 'user', 'org', or'project'. For example,
+to create a user-specific site enter:
 
 ```sh
-cd ../orgname.github.io/
-mkdocs gh-deploy --config-file ../my-project/mkdocs.yml --remote-branch master
+mkdocs gh-deploy --remote-branch master --page-type user
```

Comment:
If you're not in the working tree of the `user` or `org` repo, then how does it know which repo to push to? I don't see where that is defined or explained anywhere.